lib/Serialization/SourceInfoFormat.h

```diff
@@ -41,15 +41,15 @@ const unsigned char SWIFTSOURCEINFO_SIGNATURE[] = { 0xF0, 0x9F, 0x8F, 0x8E };
 ///
 /// See docs/StableBitcode.md for information on how to make
 /// backwards-compatible changes using the LLVM bitcode format.
-const uint16_t SWIFTSOURCEINFO_VERSION_MAJOR = 1;
+const uint16_t SWIFTSOURCEINFO_VERSION_MAJOR = 2;
 
 /// Serialized swiftsourceinfo format minor version number.
 ///
 /// Increment this value when making a backwards-compatible change that might be
 /// interesting to test for. A backwards-compatible change is one where an \e
 /// old compiler can read the new format without any problems (usually by
 /// ignoring new information).
-const uint16_t SWIFTSOURCEINFO_VERSION_MINOR = 1; // Last change: skipping 0 for testing purposes
+const uint16_t SWIFTSOURCEINFO_VERSION_MINOR = 0; // Last change: add doc comment ranges
```

Comment:
Could you use 2 here instead? we should only increment the version number.

Reply:
Sure thing. I should double-check then, I was following this:

```
/// Increment this value when making a backwards-incompatible change, i.e. where
/// an \e old compiler will \e not be able to read the new format. This should
/// be rare. When incrementing this value, reset SWIFTSOURCEINFO_VERSION_MINOR to 0.
```

I thought that, changing the size of the `DeclLocationsTableData` meant that we had to increase the major version in this case, resetting the minor version. Maybe that's not necessary though?